packages/donor/src/screens/authentication/signin/SignInScreen.tsx

```diff
@@ -54,16 +55,18 @@ export default function SignInScreen(props: SignInScreenProps) {
         />
         <div className={styles.title}>טוב לראותך שוב</div>
         <Input
+          name="sign_in"
           onChangeText={(emailContent) => {
             setEmail(emailContent);
             setEmailError("");
           }}
           value={email}
-          type={"email"}
+          type={InputType.Email}
           label={`דוא"ל`}
           errorMessage={emailError}
         />
         <Input
+          name="sign_in"
```

Comment:
password?

Reply:
The significance of the input is "sign_in", the type is "password".
This way I can communicate more effectively its' purpose in Analytics.